data/events/2024/warsaw/main.yml

```
@@ -0,0 +1,132 @@
+name: "2024-warsaw" # The name of the event. Four digit year with the city name in lower-case, with no spaces.
+year: "2024" # The year of the event. Make sure it is in quotes.
+city: "Warsaw" # The displayed city name of the event. Capitalize it.
+event_twitter: "devopsdayswaw" # Change this to the twitter handle for your event such as devopsdayschi or devopsdaysmsp
+description: "Devopsdays is coming to Warsaw!" # Edit this to suit your preferences
+ga_tracking_id: "" # If you have your own Google Analytics tracking ID, enter it here. Example: "UA-74738648-1"
+masthead_background: "dod_warsaw_banner.jpg"
+
+# All dates are in unquoted 2024-MM-DDTHH:MM:SS+TZ:TZ, like this:
+#   variable: 2019-01-04T00:00:00+02:00
+#   variable: 2019-01-05T23:59:59+02:00
+# Note: we allow 2024-MM-DD for backward compatibility, but it can lead to unexpected behaviors (like your event disappearing from the front page during your last day)
+
+startdate: 2024-11-05T00:00:00+02:00 # The start date of your event. Leave blank if you don't have a venue reserved yet.
+enddate: 2024-11-06T19:00:00+02:00 # The end date of your event. Leave blank if you don't have a venue reserved yet.
+
+# Leave CFP dates blank if you don't know yet, or set all three at once.
+cfp_date_start: 2024-03-31
+cfp_date_end: 2024-08-15
+cfp_date_announce: 2024-03-31
+
+cfp_link: "" #if you have a custom link for submitting proposals, add it here. This will control the Propose menu item as well as the "Propose" button.
+
+registration_date_start: 2024-03-31T09:00:00+02:00
+registration_date_end: 2023-11-06T19:00:00+02:00
+
+registration_closed: "" #set this to true if you need to manually close registration before your registration end date
+registration_link: "https://eventory.cc/event/devopsdays-warsaw-2024/tickets" # If you have a custom registration link, enter it here. This will control the Registration menu item as well as the "Register" button.
+
+# Location
+#
+# coordinates: "" # No longer used
+
+location: "EXPO XXI in Warsaw"
+location_address: "Ignacego Prądzyńskiego 12/14, 01-222 Warsaw"
+
+nav_elements: # List of pages you want to show up in the navigation of your page.
+  - name: propose
+    url: https://devopsdays.pl/call-for-papers-2024/
```

This will be handled by setting `cfp_link` above.
```suggestion
  - name: propose
```